packages/react-components/lib/lifts/lift-card.tsx

```diff
@@ -13,8 +13,10 @@ import {
 import React from 'react';
 import { LiftState } from 'rmf-models';
 import { doorStateToString, motionStateToString } from './lift-utils';
+import { Lift } from 'api-client';
 
 export interface LiftCardProps extends CardProps {
+  lift: Lift;
```

Comment:
As per my comment on top, once we have `LiftTableRow` internally in `lifts-app`, we won't need to modify or use these props anymore